src/components/Tags/Tags.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { axe, screen, render } from 'testUtils';
+import Tags from './';
+
+test('renders Tags with children and correctly', () => {
+  render(
+    <Tags>
+      <Tags.Tag>Hello</Tags.Tag>
+      <Tags.Tag>World</Tags.Tag>
+    </Tags>,
+  );
+  const listItems = screen.getAllByRole('listitem');
```

Comment:
```suggestion

  expect(screen.getByRole('list')).toBeInTheDocument()
  const listItems = screen.getAllByRole('listitem');
```
Use `list` to test for `<Tags />`